kconfigs/Kconfig.features

```diff
@@ -424,6 +424,11 @@ config HAS_PERIPH_USBDEV
     help
         Indicates that an USBDEV peripheral is present.
 
+config HAS_PERIPH_USBDEV_HS_UMTI
```

Comment:
```suggestion
config HAS_PERIPH_USBDEV_HS_UTMI
```

Reply:
Oops, good catch :+1:  These letter twisters are very difficult to detect.

Reply:
May I squash these fixes directly?

Reply:
I have to admit that I haven't had a look at the CI compilation log for this PR yet.

Reply:
Sure, just squash directly